lib/rules/web/ocm_console/cluster_detail.xyaml

```diff
@@ -432,4 +428,70 @@ check_install_successfully_message_missing:
     element:
         selector:
             xpath: //*[text()='Cluster installed successfully']
-        missing: true
\ No newline at end of file
+        missing: true
+check_cluster_history:
+    elements:
+        - selector:
+            xpath: //h2[text()='Cluster history']
+        - selector:
+            xpath: //div[@class='pf-l-split__item']//span[text()='Description']
+        - selector:
+            xpath: //div[@class='pf-l-split__item pf-m-fill']/input[@placeholder='Filter by Description']
+        - selector:
+            xpath: //button[text()='Download history']
+        - selector:
+            xpath: //div[@class='pf-c-table__button-content']/span[text()='Description']
+        - selector:
+            xpath: //div[@class='pf-c-table__button-content']/span[@class='pf-c-table__sort-indicator']
+        - selector:
+            xpath: //div[@class='pf-c-table__button-content']/span[text()='Severity']
+        - selector:
+            xpath: //div[@class='pf-c-table__button-content']/span[text()='Date']
+        - selector:
+            xpath: //td[@data-label='Description' and text()='Cluster installed and registered successfully']
```

Comment:
It should be better for us to speparate the history from the basic elements checking. The basic elements should be fit for every history.

Maybe there is a default history for every cluster, but still prefer this could be an individual action for it is easily for reading. For example, in your case, you use `:check_cluster_history`, it is not readable for the reader who does not know your feature.

Reply:
Updated. change the previous `:check_cluster_history` into `check_cluster_history_section_common_part` and add a new `check_default_osd_cluster_history` to cover the default history